### PR TITLE
add authoring meta to cli options

### PR DESCRIPTION
**Pull request by rouzwelt**

### CODE REVIEW

File: cli/src/cli/schema/show.rs [hunk 1]
```diff
@@ -22,17 +22,21 @@ pub struct Show {
...
     } else {
-        serde_json::to_string(&schema_json)?
-    };
-
-    crate::cli::output::output(&s.output_path, SupportedOutputEncoding::Binary, schema_string.as_bytes())
+        let schema_json = match s.schema {
+            KnownMeta::SolidityAbiV2 => schema_for!(crate::meta::solidity_abi::v2::SolidityAbi),
+            KnownMeta::InterpreterCallerMetaV1 => schema_for!(crate::meta::interpreter_caller::v1::InterpreterCallerMeta),
+            KnownMeta::OpV1 => schema_for!(crate::meta::op::v1::OpMeta),
+            KnownMeta::AuthoringMetaV1 => schema_for_value!({}),
```
Comment (thedavidmeister) [inline cli/src/cli/schema/show.rs:32]: return the error here

File: cli/src/cli/schema/show.rs [hunk 2]
```diff
@@ -22,17 +22,21 @@ pub struct Show {
...
 pub fn show(s: Show) -> anyhow::Result<()> {
-    let schema_json = match s.schema {
-        KnownMeta::SolidityAbiV2 => schema_for!(crate::meta::solidity_abi::v2::SolidityAbi),
-        KnownMeta::InterpreterCallerMetaV1 => schema_for!(crate::meta::interpreter_caller::v1::InterpreterCallerMeta),
-        KnownMeta::OpV1 => schema_for!(crate::meta::op::v1::OpMeta),
-    };
-
-    let schema_string = if s.pretty_print {
-        serde_json::to_string_pretty(&schema_json)?
+    if s.schema == KnownMeta::AuthoringMetaV1 {
```
Comment (thedavidmeister) [inline cli/src/cli/schema/show.rs:25]: dont nest an if on the same thing you are matching

File: cli/src/meta/mod.rs [hunk 3]
```diff
@@ -35,7 +37,9 @@ impl TryFrom<KnownMagic> for KnownMeta {
 pub enum ContentType {
     None,
     #[serde(rename = "application/json")]
-    Json
+    Json,
+    #[serde(rename = "application/cbor")]
```
Comment (thedavidmeister) [inline cli/src/meta/mod.rs:41]: can we test this?

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

File: cli/src/meta/mod.rs (hunk 3 above)
Comment (rouzwelt) [reply cli/src/meta/mod.rs:41]: already tested

Comment (rouzwelt) [reply cli/src/meta/mod.rs:41]: added a test in cli/src/cli/build.rs file

File: cli/Cargo.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rain_cli_meta"
-version = "0.9.0"
+version = "0.8.0"
```
Comment (thedavidmeister) [inline cli/Cargo.toml:3]: did you mean to downgrade this?

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

File: cli/Cargo.toml (hunk 4 above)
Comment (rouzwelt) [reply cli/Cargo.toml]: fixed

**Pull request merged**